examples/ECDSA.js

```diff
@@ -0,0 +1,14 @@
+/* global BROWSER_RUNTIME */
+
+let script;
```

Comment:
I thinking before on these files: why cannot we just have one or bunch `.js` files which will import all fixture files? That way we would kill all those boilerpate.

Reply:
Having loosely connected entities defined in one place is bad practice IMO:

- they will be bundled altogether even when not needed
- will make it harder to look at the scripts for the users

In general, script imports are not a problem for us at all.